9.2/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -33,6 +33,9 @@ You can also set the following mount points by passing the `-v /host:/container`
 | :----------------------- | ------------------------------------- |
 |  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
 
+
+At location `${CONTAINER_SCRIPTS_PATH}/post-init.sh` should be located script which defines all post initialization tasks. By default there is no script and only way how to get script into location is using this image as base image. In this case you should do `COPY path-to-your-script ${CONTAINER_SCRIPTS_PATH}/post-init.sh`.
```

Comment:
@rhcarvalho In your project we want to populate database after initialization. I think that would be most common use case. Loading file from local PC is better idea then creating new image for sure.